corehq/apps/sms/tasks.py

```diff
@@ -162,8 +162,10 @@ def handle_domain_specific_delays(msg, domain_object, utcnow):
 
 
 def message_is_stale(msg, utcnow):
-    oldest_allowable_datetime = \
-        utcnow - timedelta(hours=settings.SMS_QUEUE_STALE_MESSAGE_DURATION)
+    sms_stale_duration = msg.custom_metadata.get('sms_stale_after', None)
+    sms_stale_duration_hours = sms_stale_duration * 24\
+        if sms_stale_duration else settings.SMS_QUEUE_STALE_MESSAGE_DURATION
```

Comment:
I think you can just either
1. update the custom value to be in hours or
2. update SMS_QUEUE_STALE_MESSAGE_DURATION to be in days
that way this will get easier to read and you will have the same unit in both places as well.

I would suggest the first one since that gives user to fine control this even further in hours instead of days.

Reply:
Cool. I also feel 1st is better. Will be working towards it.